GW2EIEvtcParser/EIData/Buffs/Buff.cs

```diff
@@ -324,6 +324,7 @@ public bool Available(CombatData combatData)
             new Buff("Superior Rune of the Monk", SuperiorRuneOfTheMonk, Source.Gear, BuffStackType.Stacking, 10, BuffClassification.Gear, BuffImages.SuperiorRuneOfTheMonk, 93543, GW2Builds.EndOfLife),
             new Buff("Superior Rune of the Cavalier", SuperiorRuneOfTheCavalier, Source.Gear, BuffStackType.CappedDuration, 5, BuffClassification.Gear, BuffImages.SuperiorRuneOfTheCavalier),
             new Buff("Healing Glyph (Druid Runes)", HealingGlyph, Source.Gear, BuffStackType.CappedDuration, 2, BuffClassification.Gear, BuffImages.ConsumeRation),
+            new Buff("Critical (Daredevil Runes)", Critical, Source.Gear, BuffStackType.CappedDuration, 4, BuffClassification.Gear, BuffImages.Critical),
```

Comment:
BuffStackType.CappedDuration ?

Reply:
The id you provide links to a buff with BuffStackType.Stacking and a capacity of 25, did you get the wrong id?

Reply:
The rune gives this effect https://wiki.guildwars2.com/wiki/Critical_(effect) even though it's not specified on the wiki, whenever you dodge you lose the current stack and gain a new one, have i missinterpreted the behavior?